src/main/java/com/softserve/academy/studhub/entity/Question.java

```diff
@@ -30,10 +30,7 @@
     private LocalDateTime creationDate;
     @Column(name = "modified_date")
     private LocalDateTime modefiedDate;
```

Comment:
Please exclude modifiedDate from equals and hashcode and check spelling in a field name